build.env

```diff
@@ -17,7 +17,7 @@
 source ./tools/shell_functions.inc
 
 go version >/dev/null 2>&1 || fail "Go is not installed or is not in \$PATH. See https://vitess.io/contributing/build-from-source for install instructions."
-goversion_min 1.18.7 || fail "Go version reported: `go version`. Version 1.18.7+ required. See https://vitess.io/contributing/build-from-source for install instructions."
+goversion_min 1.19.3 || fail "Go version reported: `go version`. Version 1.19.3+ required. See https://vitess.io/contributing/build-from-source for install instructions."
```

Comment:
Does this _have_ to be 1.19.3+ or is any 1.19.x version acceptable?

Reply:
It requires `1.19.3+`, we don't have a thing to distinguish prod and dev environments to allow `1.19.*` in dev environments, yet.